kore/src/Kore/Log/WarnFunctionWithoutEvaluators.hs

```diff
@@ -44,9 +50,13 @@ instance SOP.HasDatatypeInfo WarnFunctionWithoutEvaluators
 
 instance Pretty WarnFunctionWithoutEvaluators where
     pretty WarnFunctionWithoutEvaluators { symbol } =
+        let Symbol { symbolAttributes } = symbol in
+        let Attribute.Symbol { klabel } = symbolAttributes in
         Pretty.vsep
             [ "No evaluators for function symbol:"
             , Pretty.indent 4 (unparse symbol)
+            , Pretty.hsep ["with label", Pretty.pretty klabel, "at location:"]
+            , Pretty.pretty (prettyPrintLocationFromAst symbol)
```

Comment:
This will give the location (in the Kore file) where the symbol is used. Let's use the location (in the K file) where the symbol is declared. We can get this from the attributes.